src/SDL_gpu_shadercross.c

```diff
@@ -312,9 +297,22 @@ static void *SDL_ShaderCross_INTERNAL_CompileUsingDXC(
     /* Non-static DxcInstance, since the functions we call on it are not thread-safe */
     IDxcCompiler3 *dxcInstance = NULL;
 
-    SDL_DxcCreateInstance(&CLSID_DxcCompiler,
-                          IID_IDxcCompiler3,
-                          (void **)&dxcInstance);
+    #if defined(SDL_PLATFORM_XBOXONE) || defined(SDL_PLATFORM_XBOXSERIES)
+    if (SDL_DxcCreateInstance == NULL) {
+        SDL_LogError(SDL_LOG_CATEGORY_APPLICATION, "%s", "DxcCreateInstance function not loaded. Did you forget to call Init?");
```

Comment:
General remark: shouldn't the library part avoid doing `SDL_LogError`, and do `SDL_SetError`.
And let the cli util do `SDL_LogError`?

Reply:
Yeah it should. I'll resolve that in another PR.